moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -73,49 +75,52 @@ const ProjectComponentsList = ({
           }
           selectedBorderColor={theme.palette.primary.main}
           additionalCollapseListItems={
-            <ListItem dense disableGutters>
-              <ListItemText
-                primary={
-                  <Button
-                    fullWidth
-                    size="small"
-                    style={{ color: theme.palette.text.primary }}
-                    startIcon={<DeleteIcon />}
-                    onClick={() => setIsDeletingComponent(true)}
-                  >
-                    Delete
-                  </Button>
-                }
-              />
-              <ListItemText
-                primary={
-                  <Button
-                    fullWidth
-                    size="small"
-                    color="primary"
-                    startIcon={<ListIcon />}
-                    onClick={onEditAttributes}
-                  >
-                    Details
-                  </Button>
-                }
-              />
-              {!isSignalComponent(component) && (
-                <ListItemText
-                  primary={
-                    <Button
-                      fullWidth
-                      size="small"
-                      color="primary"
-                      startIcon={<TimelineIcon />}
-                      onClick={onEditMap}
-                    >
-                      Map
-                    </Button>
-                  }
-                />
-              )}
-            </ListItem>
+            <Stack
+              spacing={2}
+              direction="row"
+              justifyContent="flex-end"
+              my={1}
+              // estimating alignment with zoom ListItemSecondaryAction button
+              mr={2.5}
```

Comment:
Not great, but I didn't see a way to emulate the spacing of the zoom button above without refactoring it too.

Reply:
looks good to me!